content/en/platform/corda/1.6/cenm/release-notes.md

```diff
@@ -14,6 +14,23 @@ title: Release notes
 
 # Corda Enterprise Network Manager release notes
 
+## Corda Enterprise Network Manager 1.6.1
+
+### New features and enhancements
+
+* `sshdHost` can now be specified within the `shell {}` config block for the CENM services that support Shell.
+* Added a new `host` option in the `enmListener` configuration.
+* Added a new `host` option in the `adminListener` configuration.
+* The Zone Service can now be started with a configuration file instead of command-line flags. However, the original method of starting the Zone Service has been unaffected.
```

Comment:
maybe (for zone/auth respectively)

The zone service can now be started either with a configuration file or with command line options, any existing deployments using the original method remain unaffected..

something like that maybe?

Reply:
Updated